rrfs-test/validated_yamls/templates/obtype_config/adpsfc_airTemperature_187.yaml

```diff
@@ -0,0 +1,219 @@
+     - obs space:
+         name: adpsfc_airTemperature_187
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: "@OBSFILE@"
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: hofx_adpsfc_airTemperature_187.nc4
+             allow overwrite: true
+         io pool:
+           max pool size: 1
+         observed variables: [airTemperature]
+         simulated variables: [airTemperature]
+
+       obs operator:
+         name: Composite
+         components:
+         - name: VertInterp
+           vertical coordinate: air_pressure
+           observation vertical coordinate: pressure
+           observation vertical coordinate group: MetaData
+           interpolation method: log-linear
+           variables:
+           - name: airTemperature
+
+       obs error:
+         covariance model: diagonal
+
+       obs localizations:
+         - localization method: Horizontal Gaspari-Cohn
+           lengthscale: 300e3 # orig
+
+       obs filters:
+         # ------------------
+         # airTemperature (187)
+         # ------------------
+         # Reject all obs with ObsType /= 187, QualityMarker = 4 -15
```

Comment:
That's a great question. Previously, I only applied a filter to reject obs with QualityMarker > 3, but I noticed that even after assigning errors to just 187 observations, there were still some obs with types other than 187. This didn’t make sense to me, so I added this filter to remove any observations that weren’t of type 187.

Reply:
Oh, are you suggesting changing the comment to something like "Reject all observations with QualityMarker > 3"?

Reply:
Yes, just the comment. But you are right, adpsfc contains more than 187 which I forgot. I think it is fine with the update since the obs space is for only adpsfc 187.